CMakeLists.txt

```diff
@@ -58,6 +58,8 @@ else()
 endif()
 
 option(WITH_CTEST "Enable ctest integration of tests" ON)
+option(ENABLE_LTO "Disable LTO for faster linking" ON)
```

Comment:
Maybe this flag is completely unnecessary? After all we can just set `CMAKE_INTERPROCEDURAL_OPTIMIZATION` (use default TRUE, instead of forcing TRUE in line 57?)?

Reply:
How would you switch it off using command line?

Reply:
```
if(NOT CMAKE_INTERPROCEDURAL_OPTIMIZATION)
    set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
endif()
```

or similar, see also https://github.com/scipp/scipp/blob/b3dafad98a57c79f66d4e69697aae697045b0918/CMakeLists.txt#L16-L23.